hr_payroll_commission/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Payroll Commissions and Reinbursements
+======================================
+
+This module allows you to add commission and reimbursement to payslip
+
+**Features list :**
+    * Add Worked Hours to contract.
+    * Add Hourly Rate to contract.
+    * Compute the Wage of contract, based on Worked Hours and Hourly Rate.
+    * Add commision rate, commision and reimbursement to contract.
+    * Link Expenses and Invoices to Payslip.
+
+**For further information:**
+    * Commissions and reimbursements management: http://open-net.ch/blog/la-comptabilite-salariale-suisse-avec-odoo-1/post/salaires-avec-odoo-commissions-et-notes-de-frais-78
+

```

Comment:
I splitted these modules, there was no link between commissions and expenses reimbursement.